website/docs/fun/anon-channel/modal-handler.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Modal Handler
+---
+
+This command sends anonymous messages submitted via the modal.
+
+For more information about the anonymous channel, see [this](overview) page.
+
+## Trigger
+
+**Type:** `Modal`<br />
+**Custom ID:** `\Aanon_submit\b`
+
+## Configuration
+
+See the [anonymous channel overview](overview/#configuration) for in-depth information about configuration.
```

Comment:
```suggestion
```
As there's no configuration necessary for this sub-CC, no need for this header; even less so if it just points to the main CC.

Reply:
I think you are confusing this CC with another, there is required configuration on this CC.

Reply:
I think I indeed am; ignore this review comment then.